src/test/java/org/jenkinsci/infra/tools/HyperLocalPluginManagerTest.java

```diff
@@ -0,0 +1,68 @@
+package org.jenkinsci.infra.tools;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.List;
+
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class HyperLocalPluginManagerTest {
+    private final static String pluginDir = HyperLocalPluginManagerTest.class.getResource("/git-plugin").getPath();
+    private static HyperLocalPluginManagerInit starter = new HyperLocalPluginManagerInit();
+    private static HyperLocalPluginManager pluginManager;
+    private static List<StepDescriptor> steps;
+
+    @BeforeAll
+    public static void init() {
+        pluginManager = starter.initializeHyperLocalPluginManager(pluginDir);
+        steps = pluginManager.getPluginStrategy().findComponents(StepDescriptor.class);
+    }
+
+    @Test
+    public void TotalStepsShouldBeFortySeven() {
```

Comment:
Cool.  You can also make a comment for this method showing it's testing that the `pluginStrategy` loads itself correctly.